.github/workflows/build.yml

```diff
@@ -17,22 +17,22 @@ jobs:
     steps:
       - name: Checkout PR
         if: github.event_name == 'pull_request'
-        uses: actions/checkout@v3
+        uses: actions/checkout@v4
         with:
           ref: ${{ github.event.pull_request.head.ref }}
       - name: Checkout
         if: github.event_name != 'pull_request'
         uses: actions/checkout@v3
```

Comment:
Missed this action. Need to be v4 too. 
Why do we need 2 different checkouts here?

Reply:
Version updated.
Cannot tell, why two different checkout...just bumping versions.